Parent/Parent/Conversations/ComposeReplyViewController.swift

```diff
@@ -55,8 +66,13 @@ class ComposeReplyViewController: UIViewController, ErrorViewController {
         view.backgroundColor = .named(.backgroundLightest)
 
         addCancelButton(side: .left)
-        navigationItem.rightBarButtonItem = UIBarButtonItem(title: NSLocalizedString("Send", comment: ""), style: .done, target: self, action: #selector(send))
-        navigationItem.rightBarButtonItem?.isEnabled = false
+        attachButton.accessibilityLabel = NSLocalizedString("Add Attachments", comment: "")
+        sendButton.isEnabled = false
+        navigationItem.rightBarButtonItems = [ sendButton, attachButton ]
+
+        embed(attachmentsController, in: attachmentsContainer)
+        attachmentsContainer.isHidden = true
+        attachmentsController.showOptions = showOptions(for:)
```

Comment:
I believe this is creating a retain cycle because this is strongly capturing `self`. I did verify that after composing a message and then dismissing and going back to the dashboard there is an `AttachmentCardsViewController` still in the memory graph.

Reply:
Fix should be as simple as `attachmentsController.showOptions = { [weak self] { self?.showOptions(for: $0) }`